src/main/java/org/opentripplanner/middleware/utils/ConfigUtils.java

```diff
@@ -26,13 +26,30 @@
 
     private static JsonNode envConfig;
 
+    /**
+     * Returns true only if an environment variable exists and is set to "true".
+     */
+    public static boolean getBooleanEnvVar(String var) {
+        String variable = System.getenv(var);
+        return variable != null && variable.equals("true");
+    }
+
+    /**
+     * Check if running in Travis CI. A list of default environment variables from Travis is here:
+     * https://docs.travis-ci.com/user/environment-variables/#default-environment-variables
+     */
+    public static boolean isRunningCi() {
+        return getBooleanEnvVar("TRAVIS") && getBooleanEnvVar("CONTINUOUS_INTEGRATION");
```

Comment:
The result of this should probably be cached in a static variable too.

Reply:
I'm not really sure why we're concerned with caching these results? So that we save a microsecond on evaluating the expression?